src/Resources/Auth/Auth.ts

```diff
@@ -0,0 +1,18 @@
+import AbstractResource from '../AbstractResource';
+
+export default class Auth extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = 'auth';
+        this.defaultAccess = 'guest';
```

Comment:
Unless I'm missing something this won't work?
It looks like this will do `/guest/auth/password/forgot` which is incorrect.

Reply:
Yes you're exactly right, and the API team are putting forgot and reset password behind the guest access prefix, so it will work when that change is made